test/expected/validation.out

```diff
@@ -250,8 +250,8 @@ SELECT * FROM diffix.show_settings() LIMIT 2;
 SELECT * FROM diffix.show_labels() WHERE objname LIKE 'public.test_customers%';
  objtype |         objname          |  label   
 ---------+--------------------------+----------
- table   | public.test_customers    | personal
  column  | public.test_customers.id | aid
+ table   | public.test_customers    | personal
```

Comment:
I would prefer tables be shown first. You usually care more about knowing which table is personal.